### DISCUSSION TITLE
hot keys

### DISCUSSION THREAD

**Discussion by khcarmel** [sircharlo/meeting-media-manager#228] [Ideas] [answered]
Good morning, I was wondering if it would be possible to incorporate hotkeys? Now with the hybrid meeting starting it would be nice if the background music or the years text on and off button could be on a hot key from a keyboard or something like a stream deck. Just a thought.

**Comment by sircharlo** [comment]
I like it. I'll work on something for the next release.

**Comment by sircharlo** [comment, marked as answer]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/e40c246229b47fbda242bb7a853dd427f0a6210a.

- <kbd>Alt</kbd><kbd>K</kbd>: Start/stop **K**ingdom songs for background music
- <kbd>Alt</kbd><kbd>D</kbd>: Launch media presentation mode on external **d**isplay
- <kbd>Alt</kbd><kbd>Z</kbd>: Show/hide media presentation window (which will hide/show remote **Z**oom participants on the external display)

**Reply by sircharlo** [reply]
To be released soon.

**Reply by wifilogo** [reply]
Hi Sicharlo!
Congrats for your great M3 program. Is excellent for us.
Before, when we used only VLC, with a USB stick the speaker was playing the images from the platform. VLC allows you to configure the hotkeys for each media action: Play, Stop, Forward...
Would it be possible to add and customize other hotkeys and to play the images in a public speech from the platform?
Right now, it is possible with the **_TAB_** key and the **_ENTER_** key from the keyboard (with **_Alt+Z_**), but not with a USB stick.
Thanks!

**Reply by sircharlo** [reply]
What do you mean by USB stick, exactly? An external wireless keyboard that is on the stage?

**Reply by wifilogo** [reply]
An PowerPoint USB wireless remote control.
[https://www.amazon.com/s?k=presentation+clicker+wireless+presenter+remote+usb+control+powerpoint&rh=n%3A160358011&ref=dp_bc_aui_C_4](url)
